web/src/scss/utilities/forms.scss

```diff
@@ -34,10 +34,10 @@
   }
 
   &.is-active {
-    border-color: var(--accent-color);
-    background-color: lighten($accent-color, 43.5%);
```

Comment:
since the value of `$accent-color` is the same as `$primary-color`, we can just use 

`$primary-light-color: lighten($primary-color, 43.5%);`

Reply:
Might have an issue if the color is already light, but that shouldn't block this PR.

Reply:
Yeah I think we're going to go through and remove most of the colors from kots- there too many variations to make theming easy for the vendor. 

What I'd really like to do is jus migrate to a design system and delegate theming to that. doesn't seem like kots was built with theming in mind